src/ServiceInsight.Tests/SearchViewModelTests.cs

```diff
@@ -11,20 +12,24 @@
     using Shouldly;
 
     [TestFixture]
-    public class SearchViewModelTests
+    public sealed class SearchViewModelTests : IDisposable
```

Comment:
Yeah `IDisposable` is a dinosaur. Ok I'm happy to let this go, given it's just for a test anyway.